.circleci/config.yml

```diff
@@ -487,12 +480,9 @@ jobs:
     environment:
       PR_SITE_COUNT: 10
     steps:
-      - run:
-          name: Checkout code
-          command: |
-            git clone --branch="$CIRCLE_BRANCH" --depth=1 \
```

Comment:
Shallow clones don't get "main," so we can't compare for file changes to determine if certain operations should run.